components/TokenManagePage/Media.tsx

```diff
@@ -53,7 +70,27 @@ const Media = () => {
             />
           </div>
 
-          <SaveMediaButton title={title} description={description} />
+          {
+              imageUri && (
+                  <div>
+                    <label className="font-archivo text-sm text-grey-moss-600 block mb-1">
+                      image
+                    </label>
+                    <div
+                        className="min-h-[400px] md:min-h-auto md:aspect-[571/692] relative bg-[url('/grid.svg')] bg-contain">
+                      <input
+                          ref={fileInputRef}
+                          id="media"
+                          type="file"
+                          className={`cursor-pointer z-2 size-full absolute opacity-0`}
+                          onChange={fileUpload}
+                      />
+                      <ImageBox handleImageClick={handleImageClick}/>
+                    </div>
+                  </div>
+              )
+          }
+          <SaveMediaButton/>
```

Comment:
Open Closed Principle
- actual: existing component expanded with new HTML code.
- required: new component file for `components/TokenManagePage/ImageUpload.tsx` and then update `Media.tsx` to import your new component file.